hikari/impl/buckets.py

```diff
@@ -590,18 +622,34 @@ def update_rate_limits(
                 limit_header,
                 remaining_header,
             )
-        else:
-            bucket = RESTBucket(real_bucket_hash, compiled_route)
+        except KeyError:
+            try:
+                unknown_bucket_hash = compiled_route.create_real_bucket_hash(UNKNOWN_HASH)
```

Comment:
You can't rely on the major param ids being unique, you'll get collisions between unknown ratelimit buckets for channels and guilds with the same id

Reply:
Not checked this yet but I suspect you'll l also get a ton of collisions between endpoints with no major params with this approach